pkg/dns/dns_test.go

```diff
@@ -363,14 +392,25 @@ func TestGetPendingResolveDomain(t *testing.T) {
 		{
 			name: "empty domains test",
 			args: args{
-				utCluster,
+				clusters: []*clusterv3.Cluster{
+					&utCluster,
+				},
 			},
 			want: map[string]*pendingResolveDomain{
-				"192.168.10.244": {
-					domainName: "192.168.10.244",
-					clusters:   []*clusterv3.Cluster{utCluster[0]},
+				"192.168.2.1": {
+					domainName: "192.168.2.1",
```

Comment:
This will need to be update, can you take over the commit from @nlgwcy if he agrees

Reply:
he agrees